src/Components/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import ToDoItem from '../ToDoItem/ToDoItem';
+
+const ToDoList = () => {
+    return(
+        <>
+        <ToDoItem />
+        <ToDoItem />
+        <ToDoItem />
+        <ToDoItem />
+        <ToDoItem />
+        </>

```

Comment:
Create renderList function and map mocked dummy todo list

Reply:
I created fake data and created function to display a lot components with method map().